internal/worker/worker.go

```diff
@@ -201,6 +203,46 @@ func (worker *Worker) syncVMs(ctx context.Context) error {
 		}
 	}
 
+	// Sync on-disk VMs
+	if err := worker.syncOnDiskVMs(ctx, remoteVMs); err != nil {
```

Comment:
I think we want to sync it only once upon worker registration. Don't we?

Reply:
See 72bd43eeb6e840b67146861f830a259c1b6875f1.